flink-python/pyflink/datastream/state_backend.py

```diff
@@ -956,16 +956,20 @@ class PredefinedOptions(Enum):
     determined to be beneficial for performance under different settings.
 
     Some of these settings are based on experiments by the Flink community, some follow
-    guides from the RocksDB project.
+    guides from the RocksDB project. Some configurations are enabled unconditionally (e.g.
+    setUseFsync(false), which disables syncing to storage) so they do not appear here. See the
+    documentation for the createBaseCommonDBOptions() and createBaseCommonColumnOptions() methods
```

Comment:
`createBaseCommonDBOptions` and `createBaseCommonColumnOptions` is not public in `RocksDBResourceContainer`. I think we don't need to emphasize this.

Reply:
That makes sense. I originally made this change to make PyFlink documentation match closely to the same description from the Java documentation, but mentioning the Java classes & non-public methods probably adds confusion. I made this change in the most recent commits.